src/App.vue

```diff
@@ -0,0 +1,337 @@
+<template>
+  <div id="app">
+    <div class="game-board">
+
+      <NextTileCell :nextTileNumber="baseTileNumbers.slice(-1)[0]" />
+
+      <div v-for="n in 4" class="row">
+        <div v-for="n in 4" class="cell"></div>
+      </div>
+
+      <Tile v-for="tile in tiles" :tile="tile" />
+
+    </div>
+
+    <div v-on:click="newGame" class="new-game-button">
+      New Game
+    </div>
+
+    <GameOverShroud v-show="score" :score="score" :newGame="newGame" />
+
+  </div>
+</template>
+
+<script>
```

Comment:
We were following the pattern of [single file components](https://vuejs.org/v2/guide/single-file-components.html) which have a `template` section for html, a `script` section for JS, and a `style` section for css.  The `vue-loader` in webpack knows how to read this.